src/service/rest.service.ts

```diff
@@ -8,14 +8,15 @@ export class RestService {
     return request(req, cb);
   }
 
-  public get(url: string, contentType: string = 'application/json'): Promise<any> {
+  public get(url: string, json: any = null, contentType: string = 'application/json', accept: string | string[] = [ 'application/json', 'text/plain' ], redirectCount: number = 0): Promise<any> {
```

Comment:
For code cleanliness, methods should have two or less arguments. Dyadic is preferred. Triadic is borderline messy. Four or more is in need of code cleanup.

Reply:
I would prefer a type or interface with these arguments as properties opposed to adding too many arguments.